docs/content/references/ide/debugger.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: Move Trace Debugger
+description: Debug Move execution traces directly in VS Code to understand how your code executes.
+---
+
+The [Move Trace Debugger](https://marketplace.visualstudio.com/items?itemName=mysten.move-trace-debug) extension for Visual Studio Code provides a familiar debugging interface for Move unit tests. You can step through code execution, track local variable values, and set line breakpoints to understand how your Move code executes.
+
+:::info 
+If you have the Move extension installed, you already have the Move Trace Debugger and don't need to install it separately. 
+:::
+
+The Move Trace Debugger extension is available in the Visual Studio Code Marketplace. Search for `Move Trace Debugger` in the **Extensions** view, or press <kbd>Ctrl</kbd> + <kbd>P</kbd> or <kbd>⌘</kbd> + <kbd>P</kbd> and type `ext install mysten.move-trace-debug`.
+You can also run `code --install-extension mysten.move-trace-debug` to install the extension from the command line.
+
+To use the debugger, you must also have the `sui` binary installed with the `tracing` feature flag enabled. The `sui` binary in release tarballs, Homebrew, and Chocolatey, have this feature enabled.
+See [Install Sui](../../guides/developer/getting-started/sui-install.mdx) for more information, including how to build from source.
+
+## Usage
+
+Debugging a Move unit test is a two-step process:
+
+1. Generate execution traces:
+   - Open the command palette (<kbd>⇧</kbd> + <kbd>⌘</kbd> + <kbd>P</kbd> on macOS, <kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>P</kbd> on Windows/Linux)
+   - Run the `Move: Trace Move test execution` command
+   - Find the generated trace in the `traces` directory.
+
+![Move Trace generation in the command palette](./images/trace_palette.png)
+
+2. Start debugging:
+   - Open the Move file containing your test
+   - Select **Run** -> **Start Debugging** from the menu
+   - If the file has multiple tests, select the specific test from the dropdown menu
+
+
+![Start debugging](./images/start_debugging.png)
+
+![Test selection](./images/test_selection.png)
+
+## Features
+
+Currently, the Move Trace Debugger supports basic forward debugging through test execution traces.
+Reverse debugging and watch expressions are not yet supported.
+
+### Stepping through code execution
+
+![Stepping through code execution](./images/debugger_running.png)
+
+Move Trace Debugger supports standard debugging features like step over, step into, step out, continue, and stop. You can step through normal code, and also Move macros. 
```

Comment:
Should we have a disclaimer here (or maybe we have a more general one elsewhere) that e.g., macros are supported, but are a known pain point that we are working on improving still.